subscription_manager.go

```diff
@@ -971,6 +971,10 @@ func parseCipherInterface(data interface{}, pnConf *Config, module crypto.Crypto
 					pnConf.Log.Println("v[pn_other]", v["pn_other"], v, msg)
 					decrypted, errDecryption := decryptString(module, msg)
 					if errDecryption != nil {
+                        if strings.Contains(errDecryption.Error(), "decrypt error on decode") {
+                            pnConf.Log.Println("Couldn't decode message. Asumming no encrypted message. Returning raw message...")
+                            return msg, nil
```

Comment:
Don't we want to inform user that there was a problem with message decoding?
I thought in this case we return error to the user?

Reply:
I'm afraid that our user will first check if there is any error and their program will fail in case of no encrypted message anyway what is pointless in this case. As I understood we should return `Result` type if it is possible in your implementation. Go is quite straightforward what means we cannot indicate the "something is not okay but we returning something anyway" in any way. :/

Reply:
Changed approach.